include/seqan/blast/blast_statistics.h

```diff
@@ -0,0 +1,1076 @@
+ */
+
+template <typename TMatrixSpec>
+inline double
+getH(BlastScoringScheme<Score<int, ScoreMatrix<AminoAcid, TMatrixSpec>>> const & scoringScheme)
+{
+    typedef KarlinAltschulValues<Score<int,
+        ScoreMatrix< AminoAcid, TMatrixSpec>>> TKAValues;
+    SEQAN_ASSERT(isValid(scoringScheme));
+    return TKAValues::VALUE[scoringScheme.parameterIndex][5];
+}
+
+inline double
+getH(BlastScoringScheme<Score<int, Simple>> const & scoringScheme)
+{
+    typedef KarlinAltschulValues<Score<int, Simple>> TKAValues;
+    SEQAN_ASSERT(isValid(scoringScheme));
+    return TKAValues::VALUE[scoringScheme.parameterIndex][6];
+}
+
+// ----------------------------------------------------------------------------
+// Function getAlpha
+// ----------------------------------------------------------------------------
+
+/*!
+ * @fn BlastScoringScheme#getAlpha
+ * @headerfile <seqan/blast.h>
+ * @brief Get the &alpha; value of the Karlin-Altschul parameters.
+ * @signature double getAlpha(blastScoringScheme);
+ */
+
+template <typename TMatrixSpec>
+inline double
+getAlpha(BlastScoringScheme<Score<int, ScoreMatrix<AminoAcid, TMatrixSpec>>> const & scoringScheme)
+{
+    typedef KarlinAltschulValues<Score<int, ScoreMatrix<AminoAcid, TMatrixSpec>>> TKAValues;
+    SEQAN_ASSERT(isValid(scoringScheme));
+    return TKAValues::VALUE[scoringScheme.parameterIndex][6];
+}
+
+inline double
+getAlpha(BlastScoringScheme<Score<int, Simple>> const & scoringScheme)
+{
+    typedef KarlinAltschulValues<Score<int, Simple>> TKAValues;
+    SEQAN_ASSERT(isValid(scoringScheme));
+    return TKAValues::VALUE[scoringScheme.parameterIndex][7];
+}
+
+// ----------------------------------------------------------------------------
+// Function getBeta
+// ----------------------------------------------------------------------------
+
+/*!
+ * @fn BlastScoringScheme#getBeta
+ * @headerfile <seqan/blast.h>
+ * @brief Get the &beta; value of the Karlin-Altschul parameters.
+ * @signature double getBeta(blastScoringScheme);
+ */
+
+template <typename TMatrixSpec>
+inline double
+getBeta(BlastScoringScheme<Score<int, ScoreMatrix<AminoAcid, TMatrixSpec>>> const & scoringScheme)
+{
+    typedef KarlinAltschulValues<Score<int, ScoreMatrix<AminoAcid, TMatrixSpec>>> TKAValues;
+    SEQAN_ASSERT(isValid(scoringScheme));
+    return TKAValues::VALUE[scoringScheme.parameterIndex][7];
+}
+
+inline double
+getBeta(BlastScoringScheme<Score<int, Simple>> const & scoringScheme)
+{
+    typedef KarlinAltschulValues<Score<int, Simple>> TKAValues;
+    SEQAN_ASSERT(isValid(scoringScheme));
+    return TKAValues::VALUE[scoringScheme.parameterIndex][8];
+}
+
+// ----------------------------------------------------------------------------
+// Function _lengthAdjustment
+// ----------------------------------------------------------------------------
+
+// Computes the length adjustment for E-value computation
+// Based on the public domain NCBI BLAST code by Tom Madden.
```

Comment:
I have added a more prominent notice at the top of the file, but a public domain declaration implies that I don't have to do anything, it is not a license and comes with no terms.